rest_api/restful.py

```diff
@@ -0,0 +1,139 @@
+from flask import Flask, jsonify, request, render_template
+from flask_sqlalchemy import SQLAlchemy
+from healthcheck import HealthCheck, EnvironmentDump
+from models import *
+import requests 
+import json
+import re
+
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = "postgresql://localhost:5432/crawler"
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+db = SQLAlchemy(app)
+
+
+# wrap the flask app and give a heathcheck url to make sure DB is ok
+health = HealthCheck(app, "/healthcheck")
+
+def health_database_status():
+    is_database_working = True
+    output = 'database is ok'
+
+    try:
+        session = db.session()
+        session.execute('SELECT * from backpageemail')
+    except Exception as e:
+        output = str(e)
+        is_database_working = False
+
+    print (output)
```

Comment:
Will add the 'CREATE', '.pyc' and '.DS_Store' files to the gitignore on the next commit. 

The lines you mentioned and the entire healthcheck method can be deleted, I just added them to make sure the database was working properly on my machine (for clarity, all this code can be removed):
```
health = HealthCheck(app, "/healthcheck")

def health_database_status():
    is_database_working = True
    output = 'database is ok'

    try:
        session = db.session()
        session.execute('SELECT * from backpageemail')
    except Exception as e:
        output = str(e)
        is_database_working = False

    print (output)
```

I'm not a Flask wizard either so I'll look into the pagination and try to implement that if that's possible in Flask. Currently you can pass a specific number or id as a parameter to get a subset of results.